_get_started/installation/windows.md

```diff
@@ -88,10 +88,10 @@ conda install pytorch cuda80 -c pytorch
 pip3 install torchvision
 ```
 
-#### CUDA 9.2
+#### CUDA 10.0
 
 ```bash
-conda install pytorch cuda92 -c pytorch
+conda install pytorch cuda10.0 -c pytorch
```

Comment:
Should this be `cuda100` given the first usage at the beginning of the changes? @pjh5 and @soumith?